app/data/generators/events.js

```diff
@@ -1,14 +1,25 @@
-module.exports = (faker, params) => {
+const { DateTime } = require('luxon')
+const faker = require('faker')
+faker.locale = 'en_GB'
+
+function getRandomDate(date, min = 1, max = 20) {
+  let dt = DateTime.fromJSDate(date)
+  const num = faker.random.number({ 'min': min, 'max': max })
+  return dt.plus({ days: num }).toJSDate()
+}
+
+module.exports = (params) => {
   const events = { items: [] }
+  const now = new Date(2020, 8, 12)
```

Comment:
Maybe we should abstract our fake NOW date into a separate file and then use it in all the other places to make sure the prototype works consistently with itself?

Reply:
I'll abstract this out into a `helpers/faker.js`

Reply:
I'll do the helper in a separate tranche of work as it impacts other areas of the application